.pre-commit-config.yaml

```diff
@@ -24,12 +24,6 @@ repos:
       - id: pyupgrade
         args: ["--py38-plus"]
 
-  - repo: https://github.com/myint/docformatter
```

Comment:
The docformatter just reformats docstrings to a given pattern.

I didn't see you change any docstring in this PR. What exactly is the incompatibility here?

If it is because pre-commit is trying to fix the `docs` folter, you can probably ignore that folder for that hook specifically by adding `exclude: ^docs/` to the hooks config